### ISSUE TITLE
[!] MILD ANNOUNCEMENT [!]

### ISSUE THREAD

**Issue by snaildos** [snaildos/Fifo-Browser#89]
Currently waiting on Electron to fix an issue we created. (https://github.com/electron/remote/issues/130)
This issue is REQUIRED to get fixed otherwise Fifo will not work. Fifo is currently in a hibernation state since we are waiting for that issue to get fixed before we can continue development once again!
Kind regards,
Snail

**Comment by snaildos** [comment]
Progress Tracker:
- [ ] Fix notification dialogue bug
- [ ] Fix news bug
- [ ] Ensure extension works ( #86 )

**Comment by snaildos** [comment]
Release date: 25/06/22 !

**Comment by 3oheicrw** [comment]
> Release date: 25/06/22 !

Still 1.1.0 my friend :smile:

**Comment by snaildos** [comment]
> > Release date: 25/06/22 !
> 
> Still 1.1.0 my friend 😄

Yup, we are doing some last minute testing! Trying out hardest to get this update out by this week. Trying to perfect it!

**Comment by snaildos** [comment]
Releasing tonight, Wil be closing this issue once it releases. 
Thank you for the support of our browser.

**Comment by snaildos** [comment]
@3oheicrw Our update is now live, please launch the app and wait for the app to fetch the update.
On linux, type `snap refresh`.
Thank you for the support, leave a :star: on this repo if you enjoy our work.
Thank you!
